ios/Classes/Extensions.swift

```diff
@@ -418,6 +523,36 @@ extension MapboxMaps.MapOptions {
         )
     }
 }
+extension MapboxMaps.FeaturesetFeatureId {
+    func toFLTFeaturesetFeatureId() -> FeaturesetFeatureId {
+        return FeaturesetFeatureId(
+            id: self.id,
+            namespace: self.namespace)
+    }
+}
+
+extension MapboxMaps.FeaturesetDescriptor {
+    func toFLTFeaturesetDescriptor() -> FeaturesetDescriptor {
+        return FeaturesetDescriptor(
+            featuresetId: featuresetId,
+            importId: importId,
+            layerId: layerId)
```

Comment:
@persidskiy -- this conversion from `MapboxMaps.FeaturesetDescriptor` to Flutter's `FeaturesetDescriptor` fails because `featuresetId`, `importId`, and `layerId` are private (see [here](https://github.com/mapbox/mapbox-maps-ios-internal/blob/main/mapbox-maps-ios/Sources/MapboxMaps/Interactions/FeaturesetDescriptor.swift#L15)). Is it possible to make these public or expose them in a different way?

Reply:
Also, this is why the iOS build is failing here.

Reply:
The idea is not to overexpose api that we don't need.

But I think getters can be exposed. Can you make a pr please?

Reply:
Makes sense. Yes, can make a PR 👍

Reply:
PR here: https://github.com/mapbox/mapbox-maps-ios-internal/pull/2344